internal/confmapprovider/discovery/bundle/bundle.d/receivers/oracledb.discovery.yaml.tmpl

```diff
@@ -0,0 +1,50 @@
+#####################################################################################
+#                               Do not edit manually!                               #
+# All changes must be made to associated .tmpl file before running 'make bundle.d'. #
+#####################################################################################
```

Comment:
the headers are added by the discoverybundler based on their context
```suggestion
```

Reply:
Not sure how or why these were even in here but great call out. I likely wouldnt have even noticed :P